client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -28,28 +38,51 @@ export default function OnboardingEdit() {
 				{ NO__( "Let's set up your website – it takes only a moment." ) }
 			</h2>
 
-			<label className="onboarding__question">
+			<div className="onboarding__question">
 				<span>{ NO__( 'I want to create a website ' ) }</span>
-				<SelectControl< SiteType >
-					onChange={ setSiteType }
-					options={ [
-						{ label: NO__( 'with a blog.' ), value: SiteType.BLOG },
-						{ label: NO__( 'for a store.' ), value: SiteType.STORE },
-						{ label: NO__( 'to write a story.' ), value: SiteType.STORY },
-					] }
-					value={ siteType }
-					className="onboarding__question-input"
-				/>
-			</label>
+
+				{ ! siteType && (
+					<ul className="onboarding__multi-question">
+						{ siteTypeOptions.map( ( { value, label } ) => (
+							<li key={ value }>
+								<label>
+									<input
+										checked={ siteType === value }
+										name="onboarding_site_type"
+										onChange={ handleSiteTypeChange }
+										type="radio"
+										value={ value }
+									/>
+									<span className="onboarding__multi-question-choice">{ label }</span>
+								</label>
+							</li>
+						) ) }
+					</ul>
+				) }
+				{ siteType && (
+					<div className="onboarding__multi-question">
+						<button className="onboarding__question-answered" onClick={ handleResetSiteType }>
+							{ siteType }
```

Comment:
This shouldn't just display the `siteType` (`blog`, `store`, `story`), but the label attached to it, as stored in `siteTypeOptions`. I think it'd make sense to convert the latter into an object, where the keys are the site types, and the values are the labels. This will allow us to change this line here simply into

```suggestion
							{ siteTypeOptions[ siteType ] }
```

Reply:
01cb5c6cad217c5583c9f03b08f60b78f18f4448